enum.c

```diff
@@ -4252,52 +4267,35 @@ enum_slice_when(VALUE enumerable)
 
 /*
  *  call-seq:
- *     enum.chunk_while {|elt_before, elt_after| bool } -> an_enumerator
- *
- *  Creates an enumerator for each chunked elements.
- *  The beginnings of chunks are defined by the block.
- *
- *  This method splits each chunk using adjacent elements,
- *  _elt_before_ and _elt_after_,
- *  in the receiver enumerator.
- *  This method split chunks between _elt_before_ and _elt_after_ where
- *  the block returns <code>false</code>.
- *
- *  The block is called the length of the receiver enumerator minus one.
- *
- *  The result enumerator yields the chunked elements as an array.
- *  So +each+ method can be called as follows:
+ *    chunk_while {|element, next_element| ... } -> enumerator
  *
- *    enum.chunk_while { |elt_before, elt_after| bool }.each { |ary| ... }
+ *  Each element in the returned enumerator is a 2-element array consisting of:
```

Comment:
This doesn't appear to be the case.  The value returned by the block is not included, only the chunk itself.

Reply:
Oh, wow.  Description changed.